core/src/main/java/com/scalar/db/sql/Value.java

```diff
@@ -84,6 +92,7 @@ public static Value ofNull() {
     TEXT,
     BLOB_BYTE_BUFFER,
     BLOB_BYTES,
-    NULL
+    NULL,
+    BIND_MARKER
```

Comment:
I added the `BIND_MARKER` type to `Value` that indicates the value is parameterized.